recipe/bld.bat

```diff
@@ -11,6 +11,9 @@ REM      But, use source tarball from Qt Project; projects identical as of 2018-
 
 cd Tools\qt
 
+REM Debug folder containers
+dir
+
```

Comment:
I think it should be %PYTHON% rather than `py` to conform to conda best practice...